tests/arcade_collection/analysis/test_calculate_graph_metrics.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from dataclasses import asdict
+
+from prefect import flow, task
+
+from arcade_collection.analysis import calculate_graph_metrics
+
+
+class TestCalculateGraphMetrics(unittest.TestCase):
+    def test_calculate_graph_metrics_given_unweighted_edges_calculates_metrics(self):
+        edges = [[1, 3], [2, 3], [3, 4], [4, 5], [5, 6], [5, 7], [6, 7]]
```

Comment:
Like the comment in the code, it is likely that these edges will be tuples with 3 coordinates